pkg/utils/kubernetes/reduce/reduce.go

```diff
@@ -132,6 +133,28 @@ func ReduceServices(ctx context.Context, k8sClient client.Client, services []cor
 	return nil
 }
 
+// +kubebuilder:rbac:groups="discovery.k8s.io",resources=endpointslices,verbs=list;watch
```

Comment:
Why do we need the permissions for `EndpointSlices` here? I can't see them being used

Reply:
Removed them.